autoconsent/autoconsent-impl/src/main/java/com/duckduckgo/autoconsent/impl/RealAutoconsent.kt

```diff
@@ -47,6 +52,7 @@ class RealAutoconsent @Inject constructor(
     private val settingsRepository: AutoconsentSettingsRepository,
     private val autoconsentExceptionsRepository: AutoconsentExceptionsRepository,
     private val autoconsent: AutoconsentFeature,
+    private val pixel: Pixel,
```

Comment:
Suspect I'm not doing this correctly because it's causing unit tests to fail. How should I do this differently?